src/utilities/backendValidation.ts

```diff
@@ -74,3 +75,29 @@ export default function backendValidation (fieldNames: string[], response: objec
 
   return { errorMessages, foundOnForm };
 }
+
+export default function backendValidation (fieldNames: string[], response: any): IBackendValidation {
+  if (isArray(response)) {
+    return {
+      errorMessages: [{ field: '', message: response[0] }],
```

Comment:
Do you think it's better just to display the first? Or should the errors be joined in some way?

Reply:
I used the first b/c that's how we're determining the `fieldErrors` in `getFieldErrors` method...thoughts? otherwise, might make sense to join with `;` ?

Reply:
We could map them, like `response.map(message => ({ field: '', message }))`

This gets tricky honestly because what if the backend returned an array of objects? We could handle a lot more use cases here, which I don't love. But for now, I say keep it to things we've seen "in the field"